docs_user/modules/proc_migrating-mon-from-controller-nodes-config-target-nodes.adoc

```diff
@@ -93,14 +113,56 @@ Ensure that Ceph Monitors are in a quorum and listed by the `ceph orch` command:
 ----
 +
 ----
-[ceph: root@oc0-controller-0 /]# ceph orch host ls
+$ sudo cephadm shell -- ceph orch host ls
 HOST              ADDR           LABELS          STATUS
-oc0-ceph-0        192.168.24.14  osd mon _admin
-oc0-ceph-1        192.168.24.7   osd mon _admin
-oc0-ceph-2        192.168.24.8   osd mon _admin
-oc0-controller-0  192.168.24.15  _admin mgr mon
-oc0-controller-1  192.168.24.23  _admin mgr mon
-oc0-controller-2  192.168.24.13  _admin mgr mon
+ceph-0        192.168.24.14  osd mon mgr _admin
+ceph-1        192.168.24.7   osd mon mgr _admin
+ceph-2        192.168.24.8   osd mon mgr _admin
+controller-0  192.168.24.15  _admin mgr mon
+controller-1  192.168.24.23  _admin mgr mon
+controller-2  192.168.24.13  _admin mgr mon
+----
+
+Setup a Ceph client on the first controller node that is used during the rest
+of the procedure to interact with Ceph.
+
+Setup an additional IP address on the storage network that is used to
+interact with Ceph when the first controller node is decommissioned.
```

Comment:
Are these two sentences supposed to be individual steps?

Reply:
mm no, just the same explanation of the steps below, I can make them without the extra line in the middle if the wording is good enough for you.